blog-cse/2024-05-02-content.md

```diff
@@ -0,0 +1,38 @@
+---
+title: May 2, 2024 - Content Release
+hide_table_of_contents: true
+keywords:
+  - rules
+  - log mappers
+tags: [content update]
+image: https://help.sumologic.com/img/sumo-square.png
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+NOTE: In the two weeks we are deprecating MATCH-S00408 "Fake Windows Processes" due to incorrect detection capabilites.
+
+This content release includes seventeen new rules, and two updated rules; details are enumerated below.
```

Comment:
```suggestion
This content release includes seventeen new rules and two updated rules. Details are enumerated below.
```

Reply:
@paultobiasumo, I removed the note about the deprecation because that's an internal-only thing that we don't want to announce to customers yet.